.github/workflows/server_cd_release.yml

```
@@ -93,33 +84,21 @@ jobs:
-          sudo -E /home/${{ secrets.GCE_INSTANCE_USER }}/deploy.sh # su권한으로 환경 변수 유지 및 실행
-          
-          docker image prune -af
-          docker ps -a"
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.ORACLE_INSTANCE_RELEASE_IP }}
+          username: ${{ secrets.ORACLE_INSTANCE_USER }}
+          key: ${{ secrets.ORACLE_INSTANCE_RELEASE_PRIVATE_KEY }}
+          port: ${{ secrets.ORACLE_INSTANCE_RELEASE_PORT }}
+          script: |
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)